search-api.md

```diff
@@ -230,3 +224,26 @@ A synchronous `application/json` response, of the following form:
 #### Results
 * ***Mandatory***, but can be empty
 * Is a **list of matches**, where each match has a `patient` object of the same format as the one described above for the query
+
+
+### Error handling
+The remote server should use HTTP status codes to report any errors encoundered processing the match request. Here are a list of status codes and their meanings with regards to this API:
+
+| HTTP Status Code | Constant | Description
```

Comment:
I would replace `Constant` with `Reason Phrase`, and switch the values to Title Case, [as in the RFC](http://tools.ietf.org/html/rfc7231#section-6).